providers/email_setup.rb

```diff
@@ -0,0 +1,353 @@
+#
+# Cookbook Name:: gecos-ws-mgmt
+# Provider:: email_setup
+#
+# Copyright 2018, Junta de Andalucia
+# http://www.juntadeandalucia.es/
+#
+# All rights reserved - EUPL License V 1.1
+# http://www.osor.eu/eupl
+#
+require 'json'
+require 'uri'
+
+action :setup do
+
+  begin
+
+    # Checking OS and Thunderbird
+    if new_resource.support_os.include?($gecos_os)
+
+      # Install thunderbird (in Spanish)
+      package 'thunderbird-locale-es-es' do
+        action :install
+      end
+      
+      # Xvfb is necessary for running thunderbird -CreateProfile
+      # because there is no headless mode by using MOZ_HEADLESS environment variable
+      package 'xvfb' do
```

Comment:
I think we have a brand new  package recipe. Should we move this xvfb to the package array?

Reply:
Moved!